common/men_at_arms_types/00_maa_types.txt

```diff
@@ -423,6 +423,33 @@ massive_aerial_unit = {
 	
 }
 
+riflemen = {
+	type = archers
+	
+	damage = 65
+	toughness = 10
+	pursuit = 20
+	screen = 0
+	
+	terrain_bonus = {
+		jungle = { damage = -15 }
+		forest = { damage = -10 }
+	}
+
+	counters = {
+		heavy_infantry = 1
+		heavy_cavalry = 1
+		massive_aerial = 1
+	}
+
+	buy_cost = { gold = riflemen_recruitment_cost }
+	low_maintenance_cost = { gold = riflemen_low_maint_cost }
+	high_maintenance_cost = { gold = riflemen_high_maint_cost }
+	icon = gunmen
+	
+	stack = 100
```

Comment:
You haven't added `ai_quality` from the docs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121798523-f1630c80-cc37-11eb-9b85-bcfe883d94c1.png)

</details>